src/Sylius/Bundle/CoreBundle/Storage/CartSessionStorage.php

```diff
@@ -35,7 +36,11 @@ public function __construct(
 
     public function hasForChannel(ChannelInterface $channel): bool
     {
-        return SessionProvider::getSession($this->requestStackOrSession)->has($this->getCartKeyName($channel));
+        try {
+            return SessionProvider::getSession($this->requestStackOrSession)->has($this->getCartKeyName($channel));
+        } catch (SessionNotFoundException) {
+            return false;
```

Comment:
Thanks, I need this fix.
I got `SessionNotFoundException` `There is currently no session available.`
When API recieves `OPTIONS /api/v2`